e2e-tests/pitr/run

```diff
@@ -31,7 +33,7 @@ run_recovery_check_pitr() {
 	kubectl_bin delete -f "$test_dir/conf/${restore}.yaml"
 	wait_for_running "$cluster-proxysql" 2
 	wait_for_running "$cluster-pxc" 3
-	sleep 40
+	sleep 80
```

Comment:
I'm a bit confused how it takes 80 sec to get correct data after the cluster is already online? I don't think that pure replication takes that long. Can we maybe instead of this put wait_for_cluster_consistency or something like that. It's strange.

Reply:
moved back to initial state since the waiting period has nothing to do with initial error.